api/docs/terms_of_service.md

```diff
@@ -0,0 +1,47 @@
+# Openverse API Terms of Service
+
+_Effective as of 4 May 2022_
+
+## 1. Your Agreement to the Terms
+
+Thank you for using the Openverse Catalog application programming interface (“Openverse API”).
+BY USING THE OPENVERSE API, YOU ACKNOWLEDGE THAT YOU HAVE READ, UNDERSTOOD, AND AGREED TO BE BOUND BY THESE API TERMS OF USE (hereafter, “API Terms”).
+By using the Openverse API, you also represent that you have the legal authority to accept the API Terms on behalf of yourself and any party you represent in connection with your use of the Openverse API.
+If you do not agree to the API Terms, you are not authorized to use the Openverse API.
+
+## 2. Licensed Uses and Restrictions
+
+In connection with your use of the Openverse API:
+
+  - You must adhere to all rate limits, registration requirements, and comply with all requirements in the Openverse API documentation;
+  - You must not scrape the content in the Openverse Catalog;
+  - You must not use multiple machines to circumvent rate limits or otherwise take measures to bypass our technical or security measures;
+  - You must not operate in a way that negatively affects other users of the API or impedes WordPress's ability to provide its services;
+  - You must respect and comply with the terms and conditions applicable to the content and metadata accessible through the Openverse API, including by providing proper attribution to CC-licensed works and respecting the terms and conditions of platforms hosting CC-licensed works.
+  - You must not, and the products you develop must not encourage others to, violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.
```

Comment:
```suggestion
  - You must not, and the products you develop must not, encourage others to, violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.
```

Reply:
```suggestion
  - You must not, and the products you develop must not, encourage others to violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.
```

😁

Reply:
Whoops, I think that's where I _meant_ to add the suggested comma but am too sleepy and went a few words too far! :sweat_smile: